src/main/java/picard/sam/SplitSamByNumberOfReads.java

```diff
@@ -155,10 +148,10 @@ protected int doWork() {
         CloserUtil.close(reader);
 
         if (progress.getCount() != totalReads) {
-            log.warn(String.format("The totalReads (%s) provided does not match the reads found in the " +
-                            "input file (%s). Files may not be split evenly or number of files may not " +
-                            "match what was requested. There were %s files generated each with around %s " +
-                            "reads except the last file which contained %s reads.",
+            log.warn(String.format("The totalReads (%d) provided does not match the reads found in the " +
+                            "input file (%d). Files may not be split evenly or number of files may not " +
```

Comment:
input file is a string, not number...

Reply:
Sorry, that's printing the number of reads in the input file.

Reply:
I read too quickly.... 
:+1: